### ISSUE TITLE
Keras Callback log entry wrong documented

### ISSUE THREAD

**Issue by marcj** [tensorflow/tensorflow#34965]
## URL(s) with the issue:

https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/Callback

> on_epoch_end: logs include `acc` and `loss`, and
>    optionally include `val_loss`
>    (if validation is enabled in `fit`), and `val_acc`
>    (if validation and accuracy monitoring are enabled).

and

> on_batch_end: logs include `loss`, and optionally `acc`

This is correct for the original Keras implementation. However, in tf2.keras callbacks get `accuracy` and `val_accuracy` instead of the short documented versions `acc`/`val_acc`. Either the implementation is wrong or the documentation.

**Comment by jvishnuvardhan** [comment]
@marcj Are you willing to raise a PR to correct the docs? Thanks!

**Comment by marcj** [comment]
@jvishnuvardhan does this mean this is a doc issue only and thus a breaking change compared to official/current Keras implementation?

**Comment by Jaskaran170599** [comment]
shall i make a PR? correcting "val_accuracy/accuracy" terms in doc?

> @marcj Are you willing to raise a PR to correct the docs? Thanks!

**Comment by ymodak** [comment]
The docs are now updated by the above commit. Thanks!
